src/js/components/Footer/Footer.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import { Box } from '../Box';
+
+const Footer = ({ ...rest }) => (
+  <Box
+    as="footer"
+    direction="row"
+    flex={false}
+    gap="medium"
+    justify="between"
```

Comment:
any reason not to include `align="center"`?

Reply:
Thinking about it...

Reply:
Added.